xbmc/addons/gui/GUIDialogAddonInfo.cpp

```diff
@@ -555,7 +555,7 @@ void CGUIDialogAddonInfo::OnUninstall()
   if (CDirectory::Exists("special://profile/addon_data/" + m_localAddon->ID()))
     removeData = CGUIDialogYesNo::ShowAndGetInput(CVariant{24037}, CVariant{39014});
 
-  CJobManager::GetInstance().AddJob(new CAddonUnInstallJob(m_localAddon, removeData),
+  CJobManager::GetInstance().AddJob(new CAddonUnInstallJob(m_localAddon, removeData, true),
```

Comment:
This is the unreadable code I meant when moaning about new bool params... I need to look at the method declaration to understand what the `true` means.

Reply:
will be enum class for sure

Reply:
done